packages/frint-router-component-handlers/src/createRouteHandler.js

```diff
@@ -51,7 +51,10 @@ const RouteHandler = {
   },
 
   _calculateComponentState(nextProps, appChanged = false) {
```

Comment:
maybe this method should be refactored to return earlier? i.e. check for `component` in `nextProps`, perform the logic and return. The same for the other conditions. Not a big deal though.

Reply:
👍